src/textual/widgets/_progress_bar.py

```diff
@@ -59,9 +79,15 @@ class Bar(Widget, can_focus=False):
     """The percentage of progress that has been completed."""
     _start_time: float | None
     """The time when the widget started tracking progress."""
+    color_scheme = reactive(_DEFAULT_BAR_COLOR_SCHEME)
+    """The color scheme of the widget."""
```

Comment:
```suggestion
    """The color scheme of the rule."""
```

Reply:
I hope you don't mind me sticking my nose in, but shouldn't this be "bar" rather than "rule"?

Reply:
Bar is OK.